dask/array/tests/test_creation.py

```diff
@@ -119,9 +120,36 @@ def test_arange():
     nparr = np.arange(0, -1, 0.5)
     assert_eq(darr, nparr)
 
-
-def test_arange_has_dtype():
-    assert da.arange(5, chunks=2).dtype == np.arange(5).dtype
+    # Unexpected or missing kwargs
+    with assert_raises(TypeError):
```

Comment:
In most of the codebase we use the fully qualified name `with pytest.raises(...)`.  I suggest that we stick to this for uniformity (and also fully qualified names are nice sometimes :)).